src/org/pentaho/reporting/platform/plugin/cache/AbstractReportContentCache.java

```diff
@@ -41,10 +42,19 @@ public boolean put( final String key, final IReportContent value ) {
   }
 
   @Override
+  public boolean put( final String key, final IReportContent value, Map<String, Serializable> metaData ) {
```

Comment:
If you reroute the other put to call this method with the new map as third parameter, you automatically get code coverage on here and less duplicate code.